### PR TITLE
Electronic cigarette tweaks.

### PR DESCRIPTION
**Pull request by NonQueueingMatt**
- Cleaned up electronic cigarette code slightly.
  - Electronic cigarette cartridges are now removed with alt-click.
  - Electronic cigarette cartridges now last far longer.

For some reason, e-cigs had their own snowflake transfer value, which meant that your average electronic cartridge lasted you maybe 1 minute.

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
!review

File: html/changelogs/mattatlas-ecigtweaks.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
+# Please surround your changes in  double quotes ("). It works without them, but if you use certain characters it screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - backend: "Cleaned up electronic cigarette code slightly."
+  - tweak: "Electronic cigarette cartridges are now removed with alt-click."
+  - tweak: "Electronic cigarette cartridges now last far longer."
```
Comment (FluffyGhoster) [inline html/changelogs/mattatlas-ecigtweaks.yml:60]:
```suggestion
  - code_imp: "Cleaned up electronic cigarette code slightly."
  - qol: "Electronic cigarette cartridges are now removed with alt-click."
  - qol: "Electronic cigarette cartridges now last far longer."
```

File: code/game/objects/items/weapons/ecigs.dm [hunk 2]
```diff
@@ -238,6 +192,68 @@
...
+	icon_on = "pcigon"
+	cell_type = /obj/item/cell/device/high
+
+/obj/item/clothing/mask/smokable/ecig/deluxe/get_examine_text(mob/user, distance, is_adjacent, infix, suffix)
+	. = ..()
+	if(ec_cartridge)
+		. += SPAN_NOTICE("There are [round(ec_cartridge.reagents.total_volume, 1)] unit\s of liquid remaining.")
+	else
+		. += SPAN_NOTICE("There's no cartridge connected.")
+	if(cig_cell)
```
Comment (FluffyGhoster) [inline code/game/objects/items/weapons/ecigs.dm:252]:
```suggestion

	if(cig_cell)
```

File: code/game/objects/items/weapons/ecigs.dm [hunk 3]
```diff
@@ -238,6 +192,68 @@
...
+		. += SPAN_NOTICE("The power meter shows that there's about [round(cig_cell.percent(), 5)]% power remaining.")
+	else
+		. += SPAN_NOTICE("There's no power cell connected.")
+	if(active)
```
Comment (FluffyGhoster) [inline code/game/objects/items/weapons/ecigs.dm:232]:
```suggestion

	if(active)
```

File: code/game/objects/items/weapons/ecigs.dm [hunk 4]
```diff
@@ -238,6 +192,68 @@
...
+	. = ..()
+	color = pick(ecig_colors)
+
+/obj/item/clothing/mask/smokable/ecig/util/get_examine_text(mob/user, distance, is_adjacent, infix, suffix)
+	. = ..()
+	if(ec_cartridge)
+		. += SPAN_NOTICE("There are [round(ec_cartridge.reagents.total_volume, 1)] unit\s of liquid remaining.")
+	else
+		. += SPAN_NOTICE("There's no cartridge connected.")
+	if(cig_cell)
```
Comment (FluffyGhoster) [inline code/game/objects/items/weapons/ecigs.dm:228]:
```suggestion

	if(cig_cell)
```

Comment (NonQueueingMatt) [reply code/game/objects/items/weapons/ecigs.dm:228]: epic muscle memory moment here

**Review by FluffyGhoster — APPROVED** [review]

**Review by Geevies — APPROVED** [review]

**Pull request merged**